src/constants.ts

```diff
@@ -11,3 +11,6 @@ export const DEFAULT_AUTHORIZE_TIMEOUT_IN_SECONDS = 60;
 export const DEFAULT_POPUP_CONFIG_OPTIONS: PopupConfigOptions = {
   timeoutInSeconds: DEFAULT_AUTHORIZE_TIMEOUT_IN_SECONDS
 };
+
+export const CACHE_LOCATION_MEMORY = 'memory';
+export const CACHE_LOCATION_LOCAL_STORAGE = 'localstorage';
```

Comment:
Ideally this whole mechanism would be an `enum`, but using an enum completely breaks the exports from TypeScript because we're using a mixture of default and named module exports. The only way to fix this is to introduce a breaking change, so keeping that change for a future major version.